src/app/public/modules/modal/modal.module.ts

```diff
@@ -68,8 +67,7 @@ import {
   ],
   providers: [
     MutationObserverService,
-    SkyAppWindowRef,
-    SkyThemeService
+    SkyAppWindowRef
```

Comment:
The other two providers should also be removed?

Reply:
Yep, but that work is being done in https://github.com/blackbaud/skyux-modals/pull/155
This PR is just to bring in the latest.